src/ConfidentialLedger/docs/New-AzConfidentialLedgerAADBasedSecurityPrincipalObject.md

```diff
@@ -1,100 +0,0 @@
----
-external help file:
-Module Name: Az.ConfidentialLedger
-online version: https://docs.microsoft.com/powershell/module/az.ConfidentialLedger/new-AzConfidentialLedgerAADBasedSecurityPrincipalObject
-schema: 2.0.0
----
```

Comment:
What's the purpose of removing this file? Is it expected? If yes, could you please put this change on another PR?

Reply:
I dont why this file got changed.  But i cant seem to remove the change from the PR.  any idea how to fix?

Reply:
disregard fix.  it was some corruption.  PR no longer has this change